ios/App/App/FolderPicker.swift

```diff
@@ -40,13 +40,15 @@ public class FolderPicker: CAPPlugin, UIDocumentPickerDelegate {
       didPickDocumentsAt urls: [URL]
     ){
         var items: [String] = []
-
+        let documentsPath = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true)
```

Comment:
> You should consider using the [FileManager](https://developer.apple.com/documentation/foundation/filemanager) methods [urls(for:in:)](https://developer.apple.com/documentation/foundation/filemanager/1407726-urls) and [url(for:in:appropriateFor:create:)](https://developer.apple.com/documentation/foundation/filemanager/1407693-url). which return URLs, which are the preferred format.

https://developer.apple.com/documentation/foundation/1414224-nssearchpathfordirectoriesindoma

Reply:
Thanks, here only needs `documentsPath` to be a string for later use.